### PR TITLE
Design spec for Github Integration migration

### PR DESCRIPTION
**Pull request by omgjlk**
Signed-off-by: Jesse Keating <jkeating@j2solutions.net>

### CODE REVIEW

**Comment by anne-bonny** [comment]
Merge Failed! Help can be found at https://bonnyci.org/lore/end_users/use/#handling-merge-failures

File: lore/developers/contributing/designs/zuul-github-integration-migration.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Timeline
+
+Add text here for a timeline
+
+### Zuul restart
+
+A restart of zuul is necessary to load the configuration to use the Integration, as well as to pick up some recent code changes.
+
+Add text here about performing the restart.
```
Comment (ConsoleCatzirl) [inline lore/developers/contributing/designs/zuul-github-integration-migration.md:26]: This was performed last night (2017-04-18T02:00Z)

**Review by ConsoleCatzirl — COMMENTED** [review]
From the #BonnyCI irc channel:

```text
05:58:07 +jamielennox | the easy solution is to remove the webhook_token in zuul.conf and it won't validate - but the rest was so easy we might just prompt others to switch over to integration
05:59:11 +jamielennox | the three repos not in bonny are: gearman/gearmand j2sol/giftwrap and rust-unofficial/too-many-lists
05:59:27 +jamielennox | instructions:
05:59:49 +jamielennox | 1. go to the repository on github and click the settings tab in the top right corner
06:00:14 +jamielennox | 2. Click "Webhooks" and delete the one that was created for BonnyCI
06:01:02 +jamielennox | 3. Click "Collaborators & teams" and remove access to anne-bonny (the documentation says you should put that in a robots group you can delete as well)
06:01:47 +jamielennox | 4. Go to https://github.com/integration/bonnyci and press install, accept permissions and select the repositories you want to connect
06:02:27 +jamielennox | 5. ...
06:02:29 +jamielennox | 6. Profit
```

**Comment by omgjlk** [comment]
Closing this as everything was already performed.

**Pull request closed without merging**